lightwood/mixers/helpers/default_net.py

```diff
@@ -111,6 +110,33 @@ def reset_layer_params(layer):
             else:
                 self._foward_awareness_net = self.awareness_net
 
+    def to(self, device=None, available_devices=None):
+        if device is None or available_devices is None:
+            device, available_devices = get_devices()
+
+        self.net = self.net.to(device)
+        if self.selfaware:
+            self.awareness_net = self.awareness_net.to(device)
+
+        available_devices = 1
+        if 'cuda' in str(device):
+            available_devices = torch.cuda.device_count()
+
+        if available_devices > 1:
+            self._foward_net = torch.nn.DataParallel(self.net)
```

Comment:
again, does this belong in the to method?

Reply:
Yes, it does since when we pass from one machine to another we might switch from a single GPU to multiple (and thus we have to use the `DataParallel` wrapper.

I don't really see a reason *not* to have this here. This is not the same case as with the autoencoder's `.to`.